docs/window/texture_viewer.rst

```diff
@@ -330,8 +332,12 @@ This is a powerful tool for quickly diagnosing issues and can be very useful for
 
 * ``Clipping`` will simply highlight in red any values that are below the current black point (as defined by the range control - see above), and in green any values above the white point. This can be useful in identifying invalid ranges if the range control is adjusted correctly, or in combination with a custom shader visualiser.
 
-* ``Clear before Pass`` will act as if the current target had been cleared to black right before the current pass. This can sometimes make it easier to see the results of a draw, especially if it is blending and only makes a subtle change. If the current API does not have the concept of a pass, it is defined as all the drawcalls with the same set of render targets.
+* ``Clear before Pass`` will act as if the current target had been cleared right before the current pass. This can sometimes make it easier to see the results of a draw, especially if it is blending and only makes a subtle change. If the current API does not have the concept of a pass, it is defined as all the drawcalls with the same set of render targets.
+
+   * If the current target is a render target, it is cleared to the :ref:`Alpha background` color.
 
+   * If the current target is a depth/stencil target, it is cleared to 0, cleared to 1, or unchanged, depending on the depth function specified for the selected draw.
```

Comment:
I know what you mean here from the implementation, and people can probably guess, but it'd be nice to spell out exactly how the depth is cleared based on the test: for any directional depth test (so apart from `EQUAL`/`NOT_EQUAL`/`ALWAYS`/`NEVER`) depth is cleared to a value that will always pass the first fragment written.